tests/installation/memtest.pm

```diff
@@ -17,6 +17,7 @@ sub run {
     my $self = shift;
     ensure_shim_import;
     select_bootmenu_more('inst-onmemtest', 1);
+    send_key 'f1' if (check_screen 'memtest-Fail-Safe-Mode', 10);
```

Comment:
Please add a record soft failure in this case, https://bugzilla.suse.com/show_bug.cgi?id=1210633#c4

Reply:
I have added the softfail via needle :)